guides/calculate-minfee/index.js

```diff
@@ -0,0 +1,53 @@
+const { apiClient, transactions } = require('@liskhq/lisk-client');
+
+let clientCache;
+const nodeAPIURL = 'ws://localhost:8080/ws'
+
+const getClient = async () => {
+	if (!clientCache) {
+		clientCache = await apiClient.createWSClient(nodeAPIURL);
+	}
+	return clientCache;
+};
+
+// Paste the passphrase of sender account here
+const passphrase = '';
+
+const calcMinFee = async (client) => {
+	const signedTxWithSomeFee = await client.transaction.create({
+		moduleID: 1000,
+		assetID: 0,
+		fee: BigInt(transactions.convertLSKToBeddows("1")),
+		asset: {
+			helloString: "Hello World!"
+		}
+	}, passphrase);
+
+	return client.transaction.computeMinFee(signedTxWithSomeFee);
+}
+
+const postTx = async (client, minFee) => {
+	const signedTx = await client.transaction.create({
+		moduleID: 1000,
+		assetID: 0,
+		fee: minFee-BigInt("1"),
```

Comment:
Why subtract `BigInt("1")` ?

Reply:
Good catch! I added it to verify it is in fact the minfee ;) Will remove again 👍